### PR TITLE
[bazel] Test 5.x

### PR DESCRIPTION
**Pull request by keith**
Testing for https://github.com/envoyproxy/envoy/issues/18718

### CODE REVIEW

File: .bazelrc [hunk 1]
```diff
@@ -16,8 +16,7 @@ build --color=yes
 build --workspace_status_command="bash bazel/get_workspace_status"
 build --incompatible_strict_action_env
 build --host_force_python=PY3
-build --host_javabase=@bazel_tools//tools/jdk:remote_jdk11
-build --javabase=@bazel_tools//tools/jdk:remote_jdk11
+build --java_runtime_version=remotejdk_11
```
Comment (brentleyjones) [inline .bazelrc:19]: Probably also want `--tool_java_runtime_version`, which would replace `--host_javabase`.

**Comment by brentleyjones** [comment]
Not stale

**Comment by keith** [comment]
https://github.com/envoyproxy/envoy/pull/19706#event-5983133239

**Pull request closed without merging**